openpype/hosts/resolve/api/workio.py

```diff
@@ -30,13 +29,19 @@ def save_file(filepath):
     project = get_current_project()
     name = project.GetName()
 
-    if "Untitled Project" not in name:
-        log.info("Saving project: `{}` as '{}'".format(name, file))
-        pm.ExportProject(name, filepath)
-    else:
-        log.info("Creating new project...")
-        pm.CreateProject(fname)
-        pm.ExportProject(name, filepath)
+    response = False
+    if name == "Untitled Project":
+        response = pm.CreateProject(fname)
+        log.info("New project created: {}".format(response))
+        pm.SaveProject()
+    elif name != fname:
+        response = project.SetName(fname)
+        log.info("Project renamed: {}".format(response))
+
+    if response:
+        # only export if project was saved or renamed
```

Comment:
That doesn't sound right. Does it? Technically I would still expect to be able to save into the current file (and not incrementing). If my current file is `hello_v001` then I'd still be exporting it to the same. Right?

Reply:
ah yeah, this was some dust from the development faze.